src/pye57/scan_header.py

```diff
@@ -24,12 +24,16 @@ def point_count(self):
 
     @property
     def rotation_matrix(self) -> np.array:
+        
         q = Quaternion([e.value() for e in self.node["pose"]["rotation"]])
         return q.rotation_matrix
 
     @property
     def rotation(self) -> np.array:
-        q = Quaternion([e.value() for e in self.node["pose"]["rotation"]])
+        try:
```

Comment:
thought: it would be better to use 
`try rotation = self.node["pose"]["rotation"} ... except KeyError ...`
This avoids squashing other E57Exceptions, e.g. if there is malformed data in the rotation.